README.md

```diff
@@ -1,2 +1,6 @@
 # ML-python-sklearn-nbc-cpu-training
-Jenkins X quick start for training and deploying Gaussian Naive Bayes Classification using the SciKit-Learn library
+
+
+Quickstart project for executing a Wine classifier using the SciKit-Learn framework on a CPU.
+
+This Quickstart trains the model and persists as in ONNX format. The service runtime will then serve the model on localhost where the user can then send GET requests to perform inference.
```

Comment:
I have put in a bit of info about MLOps principles and the logging etc so the user understands why we put certain steps in